packages/block-editor/src/hooks/style.js

```diff
@@ -56,6 +59,29 @@ export function getCSSVariables( styles = {} ) {
 	);
 }
 
+/**
+ * Returns the inline styles to add depending on the style object
+ *
+ * @param  {Object} styles Styles configuration
+ * @return {Object}        Flattened CSS variables declaration
+ */
+export function getInlineStyles( styles = {} ) {
+	const mappings = {
+		lineHeight: [ 'typography', 'lineHeight' ],
+		backgroundColor: [ 'color', 'background' ],
+		color: [ 'color', 'text' ],
+	};
```

Comment:
This is the important part (mapping style objects to inline styles) each time we add a new config/CSS variable, we should the right mapping here.